jax/_src/interpreters/pxla.py

```diff
@@ -2464,24 +2463,21 @@ def _to_logical_sharding(
 class MeshComputation(stages.Lowering):
   _hlo: ir.Module
   _executable: MeshExecutable | None
-  # the constants that have been hoisted out and must be passed as first args,
-  # after the tokens.
-  _const_args: list[ArrayLike]
 
   def __init__(self, name: str, hlo: ir.Module,
                const_args: list[ArrayLike],
                donated_invars: Sequence[bool], platforms: Sequence[str],
                compiler_options_kvs: tuple[tuple[str, Any], ...],
                device_assignment: xc.DeviceList | tuple[xc.Device, ...] | None,
                **compile_args):
+    self.compile_args = compile_args
```

Comment:
I prefer the arguments in the same order as in __init__ so can you please put them in the same order again?